Shared/SamplesApp.swift

```diff
@@ -31,6 +31,11 @@ struct SamplesApp: App {
 // MARK: - License
 
 extension SamplesApp {
+    /// The samples for this app.
+    static var samples: [Sample] {
+        return SamplesApp.commonSamples + SamplesApp.iOSSamples
```

Comment:
Should these be alphabetically sorted?

Reply:
Sorry, not sure I follow. At which point in the script do you mean?

Reply:
The `SamplesApp.commonSamples` and `SamplesApp.iOSSamples` can be private variables in the script, and we can expose a computed property `SamplesApp.samples` that is sorted. Does that work?

Reply:
In the template `SamplesApp+Samples.swift` file, have the following

```swift
extension SamplesApp {
    /// The common samples for this app.
    private static let commonSamples: [Sample] = [/* common_samples */]
    /// The iOS samples for this app.
    private static let iOSSamples: [Sample] = [/* ios_samples */]
    /// The samples for this app.
    static let samples = (SamplesApp.commonSamples + SamplesApp.iOSSamples).sorted { $0.name < $1.name }
}
```

And remove the extension in `Shared/SamplesApp.swift`